pkg/clients/imagecontroller/OWNERS

```diff
@@ -0,0 +1,2 @@
+reviewers:
+- build-team
```

Comment:
Does it mean individual team can only review changes touching the specific package directory?

Reply:
No, it does not. According to the [docs](https://docs.prow.k8s.io/docs/components/plugins/approve/approvers/#lgtm-label), individual team members should still be able to give out LGTMs if they want, they just shouldn't be in the reviewer pool to get automatically assigned by the OpenShift CI bot for changes outside of their directories.